### ISSUE TITLE
Mention Montenegro's "old royal capital"

### ISSUE THREAD

**Issue by axelboc** [axelboc/anki-ultimate-geography#205]
Mention **Cetinje**, the "Old Royal Capital" of Montengro, in the _Capital info_ field. See https://github.com/axelboc/anki-ultimate-geography/issues/161#issuecomment-583866082 (item no 8).

**Comment by josealberto4444** [comment]
As mentioned in #207,

> I prefer the second option a lot (which is already used in Canary Islands). Otherwise, I think we give a lot of importance to one capital and relegate others to mere curiosity that you can opt to memorize or not. Maybe that's not the intention at all, but that's how I interpreted the texts written in small font.
> 
> My propose is to put all the capitals in the capital field, separated by a comma and in a new line (a <br> in the csv might work)

I'd rather put both capitals in the Capital field as follows:

Montenegro
-
Podgorica (Official Capital)
Cetinje (Old Royal Capital)

I'd change all multiple capitals in the deck to follow this rule, but maybe we should discuss it in #207. =)

**Comment by axelboc** [comment]
Sorry, but Montenegro does not have multiple capitals. It has one: the official, constitutional capital.  Secondary capitals are provided for informational purposes only and cannot be given the same prominence as primary capitals.

A few reasons come to mind:

- This is just a guess, but I'd say few users would be interested in learning secondary capitals.
- The secondary capitals would get in the way of learning the primary capitals.
- The deck is already quite large and complex, so adding more complexity should really be avoided.
- Perhaps listing secondary capitals would affect _Capital - Country_ cards by making them too easy to learn.

**Comment by josealberto4444** [comment]
> Sorry, but Montenegro does not have multiple capitals. It has one: the official, constitutional capital. Secondary/historical capitals are provided for informational purposes only and cannot be given the same prominence as primary capitals.

Well, technically both of them are in the constitution as a kind of capitals, in the same section and with the same importance (as far as I can understand the constitution using a translator), and both of them are listed in the (Spanish) Wikipedia infobox.

>     * I'd venture to say that few users would be interested in even learning secondary capitals.

Maybe in this case, this is true, but there are other cases where all multiple capitals have the same importance officially, as for South Africa. I know this issue is about Montenegro, but as these arguments apply to others, I want to answer in general. =)

>     * The deck is already quite large and difficult, so increasing the difficulty should really be avoided.

It is completely true that it's already quite large and difficult, but so is reality. Right now, regions with multiple capitals are listed as having only one.

>     * Perhaps listing secondary capitals would affect _Capital - Country_ cards, for instance by making them too easy to learn the wrong way.

Not sure about this. As a counter argument, right now there is no card asking where is Cape Town, for example.

Also, if all capitals are listed in the capital field, you can memorize one of them and skip the others in bot Country-Capital and Capital-Country cards. However, if only one is shown in Capital-Country cards, you lose an opportunity to learn all of them at the same time.

**Comment by axelboc** [comment]
> Well, technically both of them are in the constitution as a kind of capitals, in the same section and with the same importance [...]

Wouldn't you say the word `old` makes the level of importance pretty clear? 😁 

Either way, our guideline is to follow Wikipedia's infobox, so for Montenegro, since the old capital is not listed in the English infobox, it doesn't have its place in the _Capital_ field in the English deck.

> [...] both of them are listed in the (Spanish) Wikipedia infobox.

You could have led with that! 😄 Since the Spanish infobox doesn't give levels of importance to the two capitals, then I have no objection putting them both in the _Capital_ field, like we do for the Canary Islands and now for South Africa (see my last comment in #207).

So let's go for `Podgorica y Cetiña` in the Spanish deck (without any `<br>` or parenthesised text). Feel free to provide more info in the _Capital info_ field, of course.

... Although perhaps it should be `Podgorica, Cetiña` and `Santa Cruz, Las Palmas` . I'm ashamed to say it, but until recently, I honestly thought `Santa Cruz and Las Palmas` was a single city... 😖

**Comment by aplaice** [comment]
> I'm ashamed to say it, but until recently, I honestly thought Santa Cruz and Las Palmas was a single city...

So had I, until the topic of multiple capitals came up, and was ashamed to bring it up...

I'm not sure whether a comma would help, though — I could easily imagine myself thinking that it's a way of specifying the location (something like, "Washington, D.C.") — so perhaps adding a Capital info, containing something like:

    The capital is shared between the two cities of Santa Cruz de Tenerife and Las Palmas

might be useful.

I don't have a strong opinion on whether to use a comma or an "and" in the "capital" field itself. I think that having an "and" looks slightly nicer when listing two capitals, but for three capitals, having, say `Cape Town, Pretoria and Bloemfontein` rather than `Cape Town, Pretoria, Bloemfontein` would make the field horribly long, so perhaps for consistency (and consistency with what we're doing for the flag similarity fields), it might be better to always just use a comma.

> Not sure about this. As a counter argument, right now there is no card asking where is Cape Town, for example.
> Also, if all capitals are listed in the capital field, you can memorize one of them and skip the others in both Country-Capital and Capital-Country cards. However, if only one is shown in Capital-Country cards, you lose an opportunity to learn all of them at the same time.

This brings up the question of whether we should, ideally (implementation issues aside...), have multiple Capital-Country cards, for each of the multiple capitals (so that the learner could learn the associations Cape Town - South Africa, Pretoria - South Africa and Bloemfontein - South Africa separately, rather than just learning Three capitals - South Africa). Obviously, doing this would be extremely tricky and probably overkill.

(According to [Supermemo's "Twenty rules of formulating knowledge"](https://www.supermemo.com/en/archives1990-2015/articles/20rules) learning a set of three items is acceptable, so multiple Country-Capital cards would probably not be necessary, even in the ideal case.)

**Comment by axelboc** [comment]
Yeah, the capital info for the Canary Islands looks great! I'll open an issue for it.

>This brings up the question of whether we should, ideally (implementation issues aside...), have multiple Capital-Country cards, for each of the multiple capitals.

Unfortunately, I don't believe Anki allows this.

**Comment by josealberto4444** [comment]
> Wouldn't you say the word `old` makes the level of importance pretty clear?

😱 How could I have missed that? Sorry, you are right, then.

> You could have led with that! smile Since the Spanish infobox doesn't give levels of importance to the two capitals, then I have no objection putting them both in the _Capital_ field, like we do for the Canary Islands and now for South Africa (see my last comment in #207).

You already convinced me! Now I don't want both in capital fields in Spanish either. 😂 Jokes aside, I don't really mind, let's follow Wikipedia infobox and stop discussing here.

> So let's go for `Podgorica y Cetiña` in the Spanish deck (without any `<br>` or parenthesised text). Feel free to provide more info in the _Capital info_ field, of course.

Alright, I'll do it soon.

> ... Although perhaps it should be `Podgorica, Cetiña` and `Santa Cruz, Las Palmas` . I'm ashamed to say it, but until recently, I honestly thought `Santa Cruz and Las Palmas` was a single city...

When I saw the card the first time, I had no doubt this would happen to someone. 😂

> perhaps for consistency (and consistency with what we're doing for the flag similarity fields), it might be better to always just use a comma.

I agree.

**Comment by aplaice** [comment]
### German

In none of the German Wikipedia articles on [Montenegro](https://de.wikipedia.org/wiki/Montenegro), [Podgorica](https://de.wikipedia.org/wiki/Podgorica) or [Cetinje](https://de.wikipedia.org/wiki/Cetinje) have I found any mention of Cetinje's constitutional status as the Old Royal Capital, (as opposed to the fact that it once was the capital), and they only mention that the president resides there. If the Germans (apparently) don't care about this, then arguably neither should we.

I think that the respective Norwegian articles also don't mention Cetinje's special status, but I'm far less sure.

### "Capital info" suggestions

Podgorica is the capital; Cetinje is designated as the Old Royal Capital.
or just: Cetinje is the Old Royal Capital. (?)
(similar issue for the translations)

@Erim24
DE: Podgorica ist die Hauptstadt, aber Cetinje ist der Amtssitz des Präsidenten.
or (more comprehensive): Alle wichtigen Institutionen des Montenegros haben ihren Sitz in Podgorica, mit Ausnahme des Staatspräsidenten, der in Cetinje residiert. ([basis](https://de.wikipedia.org/wiki/Podgorica)).

ES: Podgorica es la capital principal; Cetiña tiene el estatus de Antigua Capital Real.

FR: Podgoria est la capitale, pendant que Cetinje détient le titre de «capitale du trône».

@Vages
NB: Podgorica er hovedstad, men presidenten holder til i den gamle hovedstaden Cetinje. ([basis](https://no.wikipedia.org/wiki/Montenegro#Politikk_og_administrasjon))

**Comment by josealberto4444** [comment]
Shouldn't the description start like "While Podgorica&hellip;" as other similar capital info fields?

Other than that, I agree with you. =)

**Comment by axelboc** [comment]
It may be easier to not try to use "Old Royal Capital" as is, and instead turn the sentence into more of a general knowledge statement. For instance, we could drop the capital letters: "old royal capital", or use the term "historical and cultural capital". I also quite like the term "honorary capital", which is used on the English page for [Cetinje](https://en.wikipedia.org/wiki/Cetinje).

**Comment by axelboc** [comment]
Also, I don't think Podgorica should be mentioned in the capital info in English, French, German and Norwegian. In Spanish, I think it makes sense to use the formulation "While Podgorica..."

**Comment by aplaice** [comment]
> Shouldn't the description start like "While Podgorica…" as other similar capital info fields?

That would make sense — thanks!

> For instance, we could drop the capital letters: "old royal capital", or use the term "historical and cultural capital". I also quite like the term "honorary capital", which is used on the English page for Cetinje.

"Honorary capital" sounds great! I think "historical and cultural capital" isn't ideal since it doesn't reflect the "official" status of Cetinje — there are many cases where another city is a historical capital, but none others (AFAIK) where this is constitutionally recognised.

I think that in this case we could skip the "While Podgorica is the capital" part, since unlike in cases where the government is based elsewhere, it wouldn't be unclear that Cetinje isn't the actual capital:

Cetinje is an honorary capital.

(In German and Norwegian, my suggestions only mention the seat of the president, since Cetinje's special status didn't seem to be mentioned on the respective Wikipedias, so if we keep that, I think we should still have the "While Podgorica" part. Alternatively, we could just have all the versions saying that Cetinje is an honorary capital.)

**Comment by Vages** [comment]
The Norwegian translation looks good to me.

**Comment by Erim24** [comment]
I had a look at @aplaice suggestions and slightly rewrote it:
`Podgorica ist die Hauptstadt, während Cetinje Amtssitz des Präsidenten ist.`
 See #229 for a more detailed answer. And I also created a pull request (#238) with the changes by @aplaice and my changes to it.

**Comment by axelboc** [comment]
Just catching up with this, sorry. @aplaice, in #229, you wrote:

>The idea was that German Wikipedia did not seem to mention the special title/status of Cetinje [...], so arguably, the fact that it has this status is not notable for German-speakers (according to anki-ultimate-geography's policy of following the respective language Wikipedias). The seat of the presidency is mentioned and is quite important, anyway, so I proposed adding that. However, I'm entirely confident with this logic, so it might be best just to use something along the lines of "Cetinje is an honorary capital.", in all the languages, including German.

I feel the same way. To me, the fact that one version of Wikipedia doesn't mention something at all doesn't mean that this thing isn't _notable_ to the readers of that version. When the infobox in Spanish Wikipedia says that Montenegro has two capitals, it contradicts the other Wikipedias. But when German and Norwegian Wikipedia don't mention Cetinje's status of "Old Royal Capital", they don't contradict the others; they just provide less information.

In other words, I wouldn't give missing information the same importance as contradictory information. So in this case, I'd argue that it's not worth overcomplicating things and that we should just use the term "honorary capital" in all languages:

```
[EN] Cetinje is an honorary capital.
[DE] Cetinje ist eine Ehrenhauptstadt.
[ES] Cetinje es una capital honoraria.
[FR] Cetinje est une capitale honoraire.
[NB] Cetinje er en æreshovedstad.
```

**Comment by aplaice** [comment]
> In other words, I wouldn't give missing information the same importance as contradictory information. So in this case, I'd argue that it's not worth overcomplicating things and that we should just use the term "honorary capital" in all languages:

Fair enough, that does simplify things *significantly*! I'm not sure about `æreshovedstad`, though. It is what google translate gives, but searching for the word itself gives no results. It does appear that the form æres[X] for various X exists, so `æreshovedstad` would probably be understood, but it might not be what a native would use. I think the remaining translations are safe (e.g. "Ehrenhauptstadt" appears to be even used in history books).

**Comment by aplaice** [comment]
Actually, on second thoughts, since we're planning on using `Podgorica, Cetiña` as the capital, in Spanish, I think that we should mention Podgorica in the Spanish capital info, as well. It *is* implied that if Cetiña is only an honorary capital, Podgorica is a "main" capital, but it might be worth to make it explicit. Perhaps:

    Aunque Podgorica es la capital principal, Cetiña es una capital honoraria.

or even just (since they're both mentioned in the main "Capital" field):

    Podgorica es la capital principal; Cetiña es una (la?) capital honoraria.

**Comment by axelboc** [comment]
Yeah, I wasn't sure, but I think you're right.

@josealberto4444 any preference?

**Comment by josealberto4444** [comment]
I think it's perfect in the version that was merged. =)
